production/helm/loki/templates/validate.yaml

```diff
@@ -42,15 +42,18 @@
 
 {{- if and (eq (include "loki.isUsingObjectStorage" . ) "true") (not .Values.minio.enabled) }}
 {{- $bucketNames := .Values.loki.storage.bucketNames -}}
+{{- if not $bucketNames }}
```

Comment:
not sure about this, because people like to config Loki Storage like in https://github.com/grafana/loki/issues/18655#issuecomment-3140078743 without using this.